tests/savepoint/test_translate.py

```diff
@@ -24,12 +24,24 @@
 
 
 def compare_arr(computed_data, ref_data):
+    """Smooth error near zero values.
+    Inputs are arrays.
+    """
     denom = np.abs(ref_data) + np.abs(computed_data)
     compare = 2.0 * np.abs(computed_data - ref_data) / denom
     compare[denom == 0] = 0.0
     return compare
 
 
+def compare_scalar(computed_data: np.float64, ref_data: np.float64) -> np.float64:
+    """Smooth error near zero values. Scalar versions."""
+    denom = np.abs(ref_data) + np.abs(computed_data)
+    if denom == 0:
+        return np.float64(0.0)
+    else:
+        return 2.0 * np.abs(computed_data - ref_data) / denom
```

Comment:
rather than duplicating the method for scalars, you could alternatively turn the scalars into np arrays of length 1

Reply:
I though about that and got a micro-optimization reflex to not have a local allocation :D I'll change it